src/browser/webview/mattermost.js

```diff
@@ -90,7 +90,7 @@ function getUnreadCount() {
   }
 
   // mentionCount in sidebar
-  const elem = document.getElementsByClassName('badge');
+  const elem = document.querySelectorAll('#sidebar-left .badge');
```

Comment:
Is this a temporary solution? If not i think we should comment in webapp that we use the ID in desktop for scraping so people don't accidentally change it and forget this.

Reply:
Good thought! Ideally it is just a temporary solution as we're planning on changing dom scraping to communicating with the webapp directly to get data as needed.